plans/changes_src.go

```diff
@@ -156,6 +156,9 @@ type ChangeSrc struct {
 	// but have not yet been decoded from the serialized value used for
 	// storage.
 	Before, After DynamicValue
+
+	// Marked Paths
```

Comment:
It might be nice to expand this comment a little, to explain why we're storing these path value marks, and how they will be used.

Reply:
Yes, that does sound helpful! Updated.